src/lib/key_store.c

```diff
@@ -36,11 +36,14 @@
 #include "key_store_internal.h"
 #include "key_store_pgp.h"
 #include "key_store_kbx.h"
-#include "key_store_ssh.h"
 #include "packet-print.h"
 #include "packet-key.h"
 #include "packet.h"
 
+#ifdef ENABLE_SSH
+#include "key_store_ssh.h"
+#endif
+
```

Comment:
I see your point with `enable ssh`, but I don't see any point when you add this configure flag and moved it inside `Makefile.am`